pkg/osbuild/users_stage.go

```diff
@@ -39,18 +40,18 @@ func NewUsersStageOptions(userCustomizations []users.User, omitKey bool) (*Users
 	users := make(map[string]UsersStageOptionsUser, len(userCustomizations))
 	for _, uc := range userCustomizations {
 		// Don't hash empty passwords, set to nil to lock account
-		if uc.Password != nil && len(*uc.Password) == 0 {
+		if len(uc.Password.Unwrap()) == 0 {
```

Comment:
This is different behavior from the Rust we're trying to emulate where `.unwrap()` would panic if `uc.Password` is not `Some`.

Reply:
Yeah, I took this from the upstream go-optional lib, but happy to change and call it e.g. "UnwrapWithDefault" or "TakeWithDefault" or just "Take"  to make this clearer. It convinient but I agree that for rusians it might be confusing.

Reply:
What does it do in this library if the value is None? Still panic?